src/backend/src/services/ws/index.js

```diff
@@ -98,4 +99,18 @@ export default class WebSocketManager {
     });
     return players;
   }
+
+  findGame(ws) {
```

Comment:
Find game doesn't need to be an event, its just a get request. Works for now but just for future.